src/scripts/main.js

```diff
@@ -1,3 +1,358 @@
+        element.innerText = initialText;
+
+        return;
+      }
+
+      cellInput.remove();
+      element.innerText = newText;
+    });
+
+    element.addEventListener('keydown', (e) => {
+      if (e.key === 'Enter') {
+        const cellInput = document.querySelector('.cell-input');
+        const newText = cellInput.value;
+
+        if (!newText) {
+          cellInput.remove();
+          element.innerText = initialText;
+
+          return;
+        }
+
+        cellInput.remove();
+        element.innerText = newText;
+      }
+    });
+  });
+}
+
+const form = document.createElement('form');
+
+form.innerHTML = `
+  <form>
+      <label>Name:
+        <input
+          name="name"
+          type="text"
+          data-qa="name"
+          required
+        >
+      </label>
+      <label>Position:
+        <input
+          name="position"
+          type="text"
+          data-qa="position"
+          required
+        >
+      </label>
+      <label>Office:
+        <select
+          data-qa="office"
+          name="city"
+          required
+        >
+          <option>Tokyo</option>
+          <option value="singapore">Singapore</option>
+          <option value="london">London</option>
+          <option value="New York">New York</option>
+          <option value="Edinburgh">Edinburgh</option>
+          <option value="San Francisco">San Francisco</option>
+        </select>
+      </label>
+      <label>Age:
+        <input
+          name="age"
+          type="number"
+          data-qa="age"
+          required
+        >
+      </label>
+      <label>Salary:
+        <input
+          name="salary"
+          type="number"
+          data-qa="salary"
+          required
+        >
+      </label>
+      <button type="submit">Save to table</button>
+    </form>
+`;
+
+table.after(form);
+
+form.method = 'POST';
+form.action = '/';
+
+form.classList.add('new-employee-form');
+
+const saveButton = document.querySelector('button');
+
+saveButton.addEventListener('submit', (e) => {
+  e.preventDefault();
+});
+
+saveButton.addEventListener('click', () => {
+  const newRow = document.createElement('tr');
+
+  const employeeName = form.children[0].firstElementChild.value;
+  const position = form.children[1].firstElementChild.value;
+  const office = form.children[2].firstElementChild.value;
+  const age = +form.children[3].firstElementChild.value;
+  const salary = +form.children[4].firstElementChild.value;
+
+  newRow.innerHTML = `
+    <tr>
+      <td>${employeeName}</td>
+      <td>${position}</td>
+      <td>${office}</td>
+      <td>${age}</td>
+      <td>$${salary.toLocaleString('en-US')}</td>
+    </tr>
+  `;
+
+  let title;
+  let description;
+
+  const notification = document.createElement('div');
+
+  setTimeout(() => {
+    notification.remove();
+  }, 2000);
+
+  notification.dataset.qa = 'notification';
+  notification.classList.add('notification');
+
+  if (employeeName.length < 4 || age < 18 || age > 90) {
```

Comment:
consider moving numbers to consts with consistent naming and rreuse them where needed